src/networking/controllers/todos.ts

```diff
@@ -1,20 +1,52 @@
-import { Todo } from '../../types/globalTypes'
+import { ItemCheck } from '../../features/todosState'
+import { NewItem, Todo } from '../../types/globalTypes'
 import { httpService } from '../httpService'
-interface TodoResponse {
-  userId: number
-  id: number
+export interface TodoResponse {
+  description: string
+  id: string
   title: string
   completed: boolean
 }
 
+const server: string = '/tasks'
+
+function deserialization(value: TodoResponse): Todo {
+  return {
+    checked: value.completed,
+    description: value.description,
+    id: value.id,
+    title: value.title,
+  }
+}
+
 async function getTodos(): Promise<Todo[]> {
-  const response = await httpService.get<TodoResponse[]>('/todos')
-  return response.map(todo => ({
-    checked: todo.completed,
-    description: 'Description of: ' + todo.title,
-    id: todo.id,
-    title: todo.title,
-  }))
+  const response = await httpService.get<TodoResponse[]>('/tasks')
+  return response.map(todo => deserialization(todo))
+}
+
+async function postNewTodo(todo: NewItem): Promise<Todo> {
+  const { title, description } = todo
+  const response = await httpService.post<TodoResponse>(
+    server,
+    title,
+    description,
+  )
+  return deserialization(response)
+}
+
+async function modifyTodo(item: ItemCheck): Promise<Todo> {
+  const { id, title, description, toCheck } = item
+  const response = await httpService.put<TodoResponse>(
+    server + '/' + id,
```

Comment:
You can have `'/tasks/%s'` as a value in the enum I've mentioned and then do `format(Endpoints.ModifyTodo, id)`